integration-cli/requirements.go

```diff
@@ -40,7 +40,7 @@ var (
 		func() bool {
 			// Set a timeout on the GET at 15s
 			var timeout = time.Duration(15 * time.Second)
-			var url = "https://hub.docker.com"
+			var url = "http://www.baidu.com"
```

Comment:
I think it's okay to leave it as it is. But I have to be more careful if I change it again :smile: 
If in the future other contributors run into the same problem as mine, I think we can allow the user to pass the URL to check from an environment variable when running the tests.